avaframe/out3SimpPlot/outAIMEC.py

```diff
@@ -0,0 +1,547 @@
+
+    markers = ['+', 'o', 'x', '*', 's', 'd', '^', 'v', '>', '<', 'p', 'h', '.',
+               '^', 'v', '>', '<', 'p', 'h', '.']
+    mk = 0
+
+#    show flow path
+    ax1 = fig.add_subplot(111)
+#    plt.xlim([0, xlim_prof_axis])
+#    plt.ylim([0, math.ceil(max(data)+0.25)])
+#    plt.ylim([0, ymax])
+#    plt.yticks(np.arange([0, math.ceil(max(data)+0.25), ytick_increment]))
+    ax1.set_ylabel(yaxis_label, color='b', fontsize=2*fs)
+    ax1.set_xlabel(''.join(['s [m] - runout with ', str(dpp_threshold),
+                            ' kPa threshold']), color='black', fontsize=2*fs)
+    if plot_density:  # estimate 2D histogram --> create pcolormesh
+        nbins = 100
+        H, xedges, yedges = np.histogram2d(runout, data, bins=nbins)
+        H = np.flipud(np.rot90(H))
+        Hmasked = np.ma.masked_where(H == 0, H)
+        data_density = plt.pcolormesh(xedges, yedges, Hmasked, cmap=cm.Blues)
+#        data_density = plt.pcolormesh(xedges, yedges, Hmasked, cmap=cm.cool)
+        cbar = plt.colorbar(data_density, orientation='horizontal')
+        cbar.ax.set_ylabel('Counts')
+    ax2 = ax1.twinx()
+    ax2.set_ylabel('z [m]', color='g', fontsize=2*fs)
+    ax2.plot(s_path, z_path, color='green', label='path', linestyle='--', linewidth=2*lw)
+    plt.xlim([0, xlim_prof_axis])
+    plt.ylim([math.floor(min(z_path)/10)*10, math.ceil(max(z_path)/10)*10])
+    if not plot_density:
+        for k in range(len(runout)):
+            topo_name = fnames[k].split('/')[-1]
+            pfarbe = colorvar(float(k), len(runout), colorflag)
+            if k == 0:
+                ax1.plot(runout[k], data[k], marker='+',
+                         markersize=2*mks, color='g', label=topo_name)
+    #            plt.yticks(np.arange([0,5000,250]))
+                # Make the y-tick labels of first axes match the line color.
+                for tl in ax1.get_yticklabels():
+                    tl.set_color('b')
+            else:
+                ax1.plot(runout[k], data[k], label=topo_name, marker=markers[mk],
+                         markersize=mks, color=pfarbe, linewidth=lw)
+            mk = mk+1
+            if mk == len(markers):
+                mk = 1
+    plt.grid('on')
+#    plt.legend()
+#    ax1.legend(loc=0)
+#    ax2.legend(loc=0)
+
+    pro_name = fnames[0].split('/')[-3]
+    outname_fin = ''.join([outpath, '/pics/', pro_name, '_dptr',
+                           str(int(dpp_threshold)), '_', tipo, '.pdf'])
+
+    if not os.path.exists(os.path.dirname(outname_fin)):
+        os.makedirs(os.path.dirname(outname_fin))
+    fig.savefig(outname_fin, transparent=True)
+
+    plt.close(fig)
+
+    # Final result diagram - roc-plots
+    rTP = (np.array(doku[0]) / (float(doku[0][0]) + float(doku[1][0]))).astype(float)
+    rFP = (np.array(doku[2]) / (float(doku[2][0]) + float(doku[3][0]))).astype(float)
+
+
+#    rFP = (np.array(doku[2]) / (float(doku[0][0]) + float(doku[1][0]))).astype(float)
```

Comment:
remove?

Reply:
I do not use this function right now but I am planning to clean it up and use it in the next branch I make for aimec